From 6af3d25af1f0431ec1f181099cf9948eba541cc9 Mon Sep 17 00:00:00 2001
From: Suneel Garapati <sgarapati@caviumnetworks.com>
Date: Fri, 21 Dec 2018 12:05:52 -0800
Subject: [PATCH 0858/1239] 96xx: configs: add marvell vitesse phy support

This patch enables mdio command and phy driver support for
vitesse and marvell.

Change-Id: I34dc5d60ca9d858568607b4c6cb4d7f3928f5351
Signed-off-by: Suneel Garapati <sgarapati@caviumnetworks.com>
Reviewed-on: https://sj1git1.cavium.com/1473
Reviewed-by: Kostya Porotchkin <Kostya.Porotchkin@cavium.com>
Reviewed-by: Chandrakala Chavva <Chandrakala.Chavva@cavium.com>
Tested-by: Chandrakala Chavva <Chandrakala.Chavva@cavium.com>
---
 configs/octeontx2_96xx_defconfig | 3 +++
 drivers/net/octeontx2/Makefile   | 3 ++-
 2 files changed, 5 insertions(+), 1 deletion(-)

diff --git a/configs/octeontx2_96xx_defconfig b/configs/octeontx2_96xx_defconfig
index 625c5c00d4..29847185e8 100644
--- a/configs/octeontx2_96xx_defconfig
+++ b/configs/octeontx2_96xx_defconfig
@@ -70,6 +70,9 @@ CONFIG_SPI_FLASH_MACRONIX=y
 CONFIG_SPI_FLASH_SPANSION=y
 CONFIG_SPI_FLASH_STMICRO=y
 CONFIG_SPI_FLASH_WINBOND=y
+CONFIG_PHYLIB=y
+CONFIG_PHY_MARVELL=y
+CONFIG_PHY_VITESSE=y
 CONFIG_DM_ETH=y
 CONFIG_NET_OCTEONTX2=y
 CONFIG_OCTEONTX2_CGX_INTF=y
diff --git a/drivers/net/octeontx2/Makefile b/drivers/net/octeontx2/Makefile
index bfe4a0c328..21d00320a0 100644
--- a/drivers/net/octeontx2/Makefile
+++ b/drivers/net/octeontx2/Makefile
@@ -7,4 +7,5 @@
 
 
 obj-$(CONFIG_NET_OCTEONTX2) += cgx.o nix_af.o nix.o rvu_pf.o \
-				rvu_af.o rvu_common.o
+				rvu_af.o rvu_common.o octeontx_smi.o
+
-- 
2.29.0

